README.md

```diff
@@ -2,6 +2,33 @@ XRP On Ledger Settlement engine as per the proposed [Settlement RFC](https://git
 
 Note the RFC has yet to be merged and thus the implementation could change to meet the spec at a later date. 
 
+## Build the Engine
+To build this engine, issue the following command:
+
+```bash
+npm install && ./node_modules/typescript/bin/tsc && npm link
+```
+
+## Configuration
+This settlement engine provides for the following configurable settings:
+
+* **LEDGER_ADDRESS**: The XRP Ledger address that this settlement engine will listen to for incoming payments (i.e., payments made by a counterparty to the Connector account this engine is operating on behalf of). Generate a test address and secret using the [XRPL Test Faucet](https://xrpl.org/xrp-test-net-faucet.html). DEFAULT: `rGCUgMH4omQV1PUuYFoMAnA7esWFhE7ZEV`
+
+* **LEDGER_SECRET**: The XRP Ledger secret that this settlement engine will listen to for incoming payments (i.e., payments made by a counterparty to the Connector account this engine is operating on behalf of). DEFAULT: `sahVoeg97nuitefnzL9GHjp2Z6kpj`
+
+* **CONNECTOR_URL**: The base HTTP URL that this settlement engine can make API calls to in order to communicate with the Connector account this engine is operating on behalf of. DEFAULT: `http://localhost:7771`
+
+* **ENGINE_PORT**: The port that this settlement engine should bind to. DEFAULT: `3000`
+
+* **REDIS_PORT**: The port that this settlement engine should use when attempting to communicate with Redis. DEFAULT: `6379`
```

Comment:
Can you also add the REDIS_HOST option as well

Reply:
Nice catch. Fixed.

Reply:
Done!